docs/using-mina/how-to-use-zkapp.mdx

```diff
@@ -2,4 +2,38 @@
 title: How to use a zkApp
 ---
 
-Describe
+import Subhead from '@site/src/components/common/Subhead';
+
+# How to Use a zkApp
+
+<Subhead>Learn how to interact with a zero-knowledge smart contract</Subhead>
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+### Prerequisites
+
+1. Install a zkApp-compatible [Mina wallet](../using-mina/install-a-wallet).
+2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 
+
+:::note
+
+The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.
+
+:::
+
+### Instructions
+
+1. Visit the zkApp in a web browser. For example, `mycoolzkapp.com`.
+2. Interact with the zkApp as intended. For example, make a move in a game, enter in your age, and so on.
+3. Click the confirmation button to send the transaction to the Mina network.
+4. In your Mina browser wallet extension, confirm the transaction.
+5. Done! Congratulations. Your transaction is processed by the Mina network and the zkApp state is updated.
```

Comment:
```suggestion
5. Done! Congratulations. Your transaction will be processed by the Mina network and, when accepted into a block, will update the zkApp's on-chain state.
```

Reply:
We need to use future tense here b/c the transaction being sent to the network sits there until processed into a block. Only if/when it is processed into a block, will it update the zkApp account on chain.